pkg/storage/store_pool.go

```diff
@@ -327,6 +327,20 @@ func (sp *StorePool) deadReplicasGossipUpdate(_ string, content roachpb.Value) {
 	detail.deadReplicas = deadReplicas
 }
 
+// updateLocalStoreAfterRebalance is used to update local copy of target store
+// after we make an rebalance immediately.
+func (sp *StorePool) updateLocalStoreAfterRebalance(storeID roachpb.StoreID, repl *Replica) {
+	sp.detailsMu.Lock()
+	defer sp.detailsMu.Unlock()
+	detail := sp.getStoreDetailLocked(storeID)
+	// update LogicalBytes.
+	detail.desc.Capacity.LogicalBytes += repl.GetMVCCStats().Total()
+	// update WritesPerSecond
+	qps, _ := repl.writeStats.avgQPS()
+	detail.desc.Capacity.WritesPerSecond += qps
+	detail.lastUpdatedTime = sp.clock.PhysicalTime()
```

Comment:
I don't think we should actually update this, since it would mess up calculations of whether the store is live or dead. This is only meant to be modified when new gossip is received from the store.

Reply:
My mistake.

Reply:
Done.